frontend/src/pages/instance/components/EditorLink.vue

```diff
@@ -43,15 +43,11 @@ export default {
             required: true
         }
     },
-    data () {
-        return {
-            canUseImmersiveEditor: false
-        }
-    },
     computed: {
         ...mapState('account', ['team', 'teamMembership']),
         isImmersiveEditor () {
-            return this.canUseImmersiveEditor && SemVer.satisfies(SemVer.coerce(this.instance?.meta?.versions?.launcher), '>=2.3.1')
+            return SemVer.satisfies(SemVer.coerce(this.instance?.meta?.versions?.launcher), '>=2.4.1') &&
+                SemVer.satisfies(SemVer.coerce(this.instance?.meta?.versions?.['node-red']), '>=4.0')
```

Comment:
Only question is will this match the existing beta releases?

Reply:
Will we be using beta releases in production? I haven't tested the outcome with a beta release but can do so if needed

Reply:
quick test:
```sh
satisfies('3.1.0-beta', '>=2.3.1') true
satisfies('3.1.0-beta', '>=3.2') false
satisfies('3.1.0-beta', '>=4.0') false 
```

Reply:
@cstns it is the 4.0 beta that we care about:
```
satisfies('4.0.0-beta.4', '>=4.0') => false
```

However, as this code uses `coerce` as well, then:

```
satisfies(c.coerce('4.0.0-beta.4'), '>=4.0') => true
```

So I think we're okay.